src/desktop/users.desktop

```diff
@@ -9,7 +9,7 @@ X-SuSE-YaST-Call=users
 X-SuSE-YaST-Group=Security
 X-SuSE-YaST-Argument=
 X-SuSE-YaST-RootOnly=true
-X-SuSE-YaST-AutoInst=all
+X-SuSE-YaST-AutoInst=configure
```

Comment:
I think we should set it back to "all" because this module can still be used in an installed system for writing user settings.